ontology/occo.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: ontology_detail
+id: occo
+title: Occupation Ontology
+browsers:
+- title: BioPortal Ontology Browser
+  label: BioPortal
+  url: https://bioportal.bioontology.org/ontologies/OCCO?p=classes
+contact:
+  email: zhengj2007@gmail.com
+  github: zhengj2007
+  label: Jie Zheng
+  orcid: 0000-0002-2999-0103
+description: The Occupation Ontology (OccO) is an ontology in the domain of human occupations. OccO is an initial ontological presentation of information taken from a standardized occupation taxonomy, the US Standard Occupational Classification (SOC) as enhanced by the O*Net system.
+domain: information
+homepage: https://github.com/Occupation-Ontology/OccO
+issue_requested: 2428
+license:
+  label: CC BY 4.0
+  url: https://creativecommons.org/licenses/by/4.0/
+preferredPrefix: OCCO
+products:
+- id: occo.owl
+repository: https://github.com/Occupation-Ontology/OccO
+review:
+  date: 2023
```

Comment:
This is an old flag which is meaningless now; I think this should be deleted from all records:

```
rep "review:" ontology/*.md
ontology/bfo.md:review:
ontology/chebi.md:review:
ontology/doid.md:review:
ontology/go.md:review:
ontology/obi.md:review:
ontology/occo.md:review:
ontology/pato.md:review:
ontology/po.md:review:
ontology/pr.md:review:
ontology/xao.md:review:
ontology/zfa.md:review:
```

cc @deepakunni3 maybe make an issue to remove all this and remove the element from the metadata schema as well.

Reply:
Removed:
review:
  date: 2023

Reply:
Thank you!